Gantt: Add Overview (#7631)
DevExpress/devextreme-documentation: concepts/05 UI Components/Gantt/05 Overview/00 Overview.md

> DevExtreme Gantt is an interactive component that helps plan tasks on a timeline and visualize dependencies between them. The component displays tasks in customizable columns alongside the timeline view and supports column filtering operations.

Reviewer: DevExtreme Gantt is a project management tool to schedule and plan projects. Gantt helps plan tasks on a timeline and visualize dependencies between them. 

We should highlight that Gantt suits for hierarchical tasks

Reviewer: Please mention resources, resource assignments

Reviewer: quotes from other landing to explain what the visualization for:
> can see the project start/end date, tasks progress, dependencies between tasks, milestones and other components of a project, at a glance.

>  It helps assess how long a project should take, determine the resources needed, manage the dependencies between tasks, and plan the order in which the tasks should be completed.

> DevExtreme Gantt is a project management tool that helps schedule and plan projects. The component displays tasks on a timeline and visualizes dependencies between hierarchical tasks. Gantt can highlight task progress, milestones, and resource/team assignments. Alongside the timeline, the component displays tasks in a configurable column view which supports filtering and sorting operations.

Reviewer 2: ```suggestion
DevExtreme Gantt is a project management component for scheduling and planning projects. It displays tasks on a timeline, shows dependencies between hierarchical tasks, and highlights task progress, milestones, and resource or team assignments. A configurable column view alongside the timeline supports filtering and sorting.
```

Author: ```suggestion
DevExtreme Gantt is a project management component for scheduling and planning projects. It displays tasks on a timeline, visualizes dependencies between hierarchical tasks, and highlights task progress, milestones, and resource or team assignments. A configurable column view alongside the timeline supports filtering and sorting.
```
